lte/gateway/c/oai/tasks/amf/amf_app_ue_context_and_proc.h

```diff
@@ -755,4 +748,23 @@ void amf_app_handle_cm_idle_on_ue_context_release(
 void ue_context_release_command(
     amf_ue_ngap_id_t amf_ue_ngap_id, gnb_ue_ngap_id_t gnb_ue_ngap_id,
     Ngcause ng_cause);
+int amf_registration_run_procedure(amf_context_t* amf_context);
+int amf_proc_registration_complete(
+    amf_ue_ngap_id_t ue_id, bstring smf_msg_p, int amf_cause,
+    const amf_nas_message_decode_status_t status);
+
+// Finite state machine handlers
+int state_handle_message_ue_1(
+    m5gmm_state_t cur_state, int event, SMSessionFSMState session_state,
+    ue_m5gmm_context_s* ue_m5gmm_context, amf_context_t* amf_context);
+int state_handle_message_ue_2(
+    m5gmm_state_t, int, SMSessionFSMState, ue_m5gmm_context_s*,
+    amf_ue_ngap_id_t, bstring, int, amf_nas_message_decode_status_t);
+int state_handle_message_ue_3(
+    m5gmm_state_t, int event, SMSessionFSMState, ue_m5gmm_context_s*,
+    amf_ue_ngap_id_t);
+int state_session_handle_message_1(
```

Comment:
we need to find better names for these before porting to master branch.

Reply:
Changed the API names accordingly.